tests/schemas/transaction.schema.js

```diff
@@ -71,8 +71,21 @@ const postTransactionSchema = {
 	message: Joi.string().valid('Transaction payload was successfully passed to the network node').required(),
 };
 
+const pendingTransactionSchemaVersion5 = {
+	id: Joi.string().required(),
+	moduleAssetId: Joi.string().required(),
+	moduleAssetName: Joi.string().required(),
+	fee: Joi.string().required(),
+	nonce: Joi.string().optional(),
```

Comment:
Isn't `nonce` mandatory?
```suggestion
	nonce: Joi.string().required(),
```